python/mxnet/ndarray/contrib.py

```diff
@@ -28,7 +28,7 @@
 except ImportError:
     pass
 
-__all__ = ["rand_zipfian"]
+__all__ = ["rand_zipfian", "foreach", "while_loop"]
```

Comment:
need to update docs/api/python/ndarray&symbol/contrib.md, too

Reply:
Updated